saviSimulation/savi/jason_processing/UaVBehavior.java

```diff
@@ -28,9 +28,9 @@
 	 * @param type
 	 * @param initialPosition
 	 */
-	public UaVBehavior(String id, String type, PVector initialPosition, double reasoningCyclePeriod) {	
+	public UaVBehavior(String id, String type, PVector initialPosition, double reasoningCyclePeriod, double sensorsErrorProb, double sensorsErrorStdDev, int seed) {	
 		// Initialize data values
-		super(id, type, initialPosition,reasoningCyclePeriod);
+		super(id, type, initialPosition,reasoningCyclePeriod, sensorsErrorProb, sensorsErrorStdDev, seed);
```

Comment:
I don't think the random seed should be passed as a parameter here.
In fact I would be in favour of having a single static Random object for the class. Otherwise the different objects will all have the same sequence of sensor malfunction (same random seed => same values drawn at each simulation round)